### DISCUSSION TITLE
.Net versions supported

### DISCUSSION THREAD

**Discussion by ZavalichiR** [OpenCover/opencover#1016] [Q&A]
Hello!
Which are the latest .Net versions suported? (.Net x, .Net core and .Net framework)
Are you planning to support .Net 6 version?

**Comment by sawilde** [comment]
Windows only
All full frameworks are supported from .net 2 to .net 4.8

From the next release (another alert to address)
We are only supporting 2.2 and 3.1 as far as .net core is concerned
We will also support .net 5

Any preview versions of .net are not supported but may work; this is due prior experience in trying to support preview versions
